### PR TITLE
Console - application general migration - fourth part

### PR DESCRIPTION
**Pull request by jansobczynski**
## Issue

https://gravitee.atlassian.net/browse/APIM-XXX

## Description

Migrating application general tab - include ARCHIVED status functionality

## Additional context

Screenshot:
Screenshot 2023-11-22 at 12 30 14 PM

---

📚&nbsp;&nbsp;View the storybook of this branch [here](https://612657caa8e859003a8a6430-qbjlldfnde.chromatic.com)

### CODE REVIEW

File: gravitee-apim-console-webui/src/management/application/details/general/general-ng/application-general.component.html [hunk 1]
```diff
@@ -22,20 +22,20 @@
         <div class="details-card__header__info-inputs__first-row">
           <mat-form-field class="details-card__header__info-inputs__first-row__name-field">
             <mat-label>Application name</mat-label>
-            <input formControlName="name" matInput type="text" maxlength="50" required />
+            <input formControlName="name" matInput type="text" maxlength="50" required [readonly]="isReadOnly" />
```
Comment (ThibaudAV) [inline gravitee-apim-console-webui/src/management/application/details/general/general-ng/application-general.component.html]: I think it's better to use the `disabled` state provided by the form control in these situations.

Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/management/application/details/general/general-ng/application-general.component.html]: see how it's done in the other forms ;)

Comment (jansobczynski) [reply gravitee-apim-console-webui/src/management/application/details/general/general-ng/application-general.component.html]: Done

Comment (ThibaudAV) [reply gravitee-apim-console-webui/src/management/application/details/general/general-ng/application-general.component.html]: haha no  📲

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**